rules/coding-style/tests/Rector/PostInc/PostIncDecToPreIncDecRector/Fixture/skip_as_array_key.php.inc

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Rector\CodingStyle\Tests\Rector\PostInc\PostIncDecToPreIncDecRector\Fixture;
+
+class SkipAsArrayKey
+{
+    public function run($value = 1)
+    {
+        $x[$value++];
+        $x[$value--];
+    }
+}
```

Comment:
This can refactorable like described in the issue:

```diff
-$values[$key++];
+$values[$key];
+++$key;
```

Reply:
implemented.